app/_src/explore/gateway-api.md

```diff
@@ -2,14 +2,18 @@
 title: Kubernetes Gateway API
 ---
 
-{{site.mesh_product_name}} supports configuring [built-in gateway](/docs/{{ page.version }}/explore/gateway) using [Kubernetes Gateway API](https://gateway-api.sigs.k8s.io/).
+{{site.mesh_product_name}} supports [Kubernetes Gateway API](https://gateway-api.sigs.k8s.io/)
+for configuring [built-in gateway](/docs/{{ page.version }}/explore/gateway)
+as well as traffic routing using the experimental
+[GAMMA](https://gateway-api.sigs.k8s.io/contributing/gamma/)
+[routing spec](https://gateway-api.sigs.k8s.io/geps/gep-1426/).
 
 ## Installation
 
 {% warning %}
 [Kubernetes Gateway API](https://gateway-api.sigs.k8s.io/) is still beta, therefore {{site.mesh_product_name}}'s integration provides the same level of stability.
 
-Gateway API is not supported in multi-zone. To use the builtin Gateway, you need to use the [`MeshGateway` resources](/docs/{{ page.version }}/explore/gateway).
+Gateway API `Gateways` aren't supported in multi-zone. To use the builtin Gateway, you need to use the [`MeshGateway` resources](/docs/{{ page.version }}/explore/gateway).
```

Comment:
Maybe I'm always confused about this Gateway / Gateway API / API Gateway - should we just link to docs - https://gateway-api.sigs.k8s.io/concepts/api-overview/#gateway ?

Reply:
yeah let me add a couple links